src/utils/getTemplateParams.js

```diff
@@ -88,9 +100,15 @@ const getSwalInput = (templateContent) => {
   if (input) {
     showWarningsForAttributes(input, ['type', 'label', 'placeholder', 'value'])
     result.input = input.getAttribute('type') || 'text'
-    result.inputLabel = input.getAttribute('label') || ''
-    result.inputPlaceholder = input.getAttribute('placeholder') || ''
-    result.inputValue = input.getAttribute('value') || ''
+    if (input.getAttribute('label')) {
+      result.inputLabel = input.getAttribute('label')
+    }
+    if (input.getAttribute('placeholder')) {
+      result.inputPlaceholder = input.getAttribute('placeholder')
+    }
+    if (input.getAttribute('value')) {
```

Comment:
@limonte Better to use `input.hasAttribute` or else some cases won't work as expected.. e.g. `<input type="text" value=""/>`